examples/ticketing-connector/main.go

```diff
@@ -0,0 +1,521 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+
+	"github.com/google/uuid"
+
+	"github.com/synqly/go-sdk/client/engine"
+	engineClient "github.com/synqly/go-sdk/client/engine/client"
+	mgmt "github.com/synqly/go-sdk/client/management"
+	mgmtClient "github.com/synqly/go-sdk/client/management/client"
+)
+
+var consoleLogger = log.New(os.Stdout, "INFO: ", log.Ldate|log.Ltime)
+
+type App struct {
+	Tenant *Tenant
+	Config *Config
+}
+
+// NewApp instantiates a new App
+func NewApp(config *Config) *App {
+	return &App{
+		Config: config,
+	}
+}
+
+// A Tenant object represents one of your customers, as well as the state you
+// would maintain for them in your application.
+type Tenant struct {
+	// ID: A unique identifier for one of your customers.
+	ID string
+	// SynqlyAccountId: The ID of the Synqly Account in which Integrations for
+	// this tenant will be created. This example creates a new Account for every
+	// Tenant, but it would also be possible to use the same Account for all Tenants.
+	SynqlyAccountId string
+	// SynqlyClient: A cached management client, used to manage Integrations.
+	SynqlyClient *mgmtClient.Client
+	// TicketClient: A cached engine client, used to log events to a third-party
+	// logging Provider by way of an Integration.
+	TicketClient *engineClient.Client
+}
+
+// NewTenant initializes a Synqly Account for a given Tenant. This example
+// creates a new Account for every Tenant to keep their credentials isolated.
+//
+// Returns an error if a tenant with the same ID already exists or if a Synqly
+// Account cannot be created for the Tenant.
```

Comment:
```suggestion
// NewTenant is used in a multi-tenant backend to create tenants. This would
// normally contain the logic you need to add a customer. When using Synqly,
// part of this logic should include creating an Account for them, and that is 
// what's done here.
//
// Returns an error if a tenant with the same ID already exists or if a Synqly
// Account cannot be created for the Tenant.
```